esy-lib/Path.rei

```diff
@@ -16,7 +16,8 @@ let append : (t, t) => t;
 
 let ofString : string => result(t, [> |`Msg(string)]);
 let current : unit => Run.t(t);
-let user : unit => Run.t(t);
+let homeDir : unit => Run.t(t);
```

Comment:
Let's rename to `homePath`.

Reply:
Let's also make it `val home : t` (that means module will be side effectful and query environment and fail if something isn't right but this will simplify the consumer code).